config/routes.rb

```diff
@@ -4,4 +4,5 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  post 'webhooks/process'
```

Comment:
@shrudhinvraj - We would need a webhook source as well so that we can extend the same for other requests in future. Something like 👇🏻 
```suggestion
  post 'webhooks/:webhook_source',  controller: :webhooks, action: :process 
```